src/otx/algo/detection/heads/ssd_head.py

```diff
@@ -11,15 +11,16 @@
 
 from otx.algo.detection.heads.anchor_head import AnchorHead
 from otx.algo.detection.heads.base_sampler import PseudoSampler
-from otx.algo.detection.heads.custom_anchor_generator import SSDAnchorGeneratorClustered
+from otx.algo.detection.heads.custom_anchor_generator import AnchorGenerator
 from otx.algo.detection.heads.delta_xywh_bbox_coder import DeltaXYWHBBoxCoder
 from otx.algo.detection.heads.max_iou_assigner import MaxIoUAssigner
 from otx.algo.detection.losses.cross_entropy_loss import CrossEntropyLoss
 from otx.algo.detection.losses.weighted_loss import smooth_l1_loss
 from otx.algo.detection.utils.utils import multi_apply
 
 if TYPE_CHECKING:
-    from mmengine.config import ConfigDict, InstanceData
+    from mmengine.config import InstanceData
```

Comment:
If this PR is merged, is this the only mm-related thing left?

Reply:
InstanceData and load_checkpoint I think.